README.md

```diff
@@ -9,34 +9,20 @@ You are supposed to build a website which shows random "fortune" messages, just
 like the one you find in fortune cookies. Note that our fortune messages also
 include a title.
 
-Each fortune message may receive anonymous comments, the comment must be at
-least 10 character long and at most 255 characters long.
+Each fortune message may receive comments, the comment must be at least 10
```

Comment:
Eu colocaria:

Each fortune message may receive comments **from users**, the comment must be at least 10

só pra saberem rsrs